frontend/src/pages/chat/Chat.tsx

```diff
@@ -94,17 +94,10 @@ const Chat = () => {
     }, [appStateContext?.state.chatHistoryLoadingState])
 
     const getUserInfoList = async () => {
-        if (!AUTH_ENABLED) {
-            setShowAuthMessage(false);
-            return;
-        }
-        const userInfoList = await getUserInfo();
-        if (userInfoList.length === 0 && window.location.hostname !== "127.0.0.1") {
-            setShowAuthMessage(true);
-        }
-        else {
-            setShowAuthMessage(false);
-        }
+      const userInfoList = await getUserInfo();
+      if (userInfoList.length === 0 && window.location.hostname == "127.0.0.1") {
```

Comment:
This changes logic for localhost users in a way I don't think we want

Reply:
Is the update to this method changing the expected behavior? This particular check was already in place. Should this be changed? If so how?

Reply:
The change from `!==` to `==` in the hostname check. As currently written it will only update when the hostname is localhost.

Reply:
Oh! Thanks for catching that! That was updated to `==` for testing only. I'll revert back to `!==` as intended.

Reply:
Fixed in update.